assets/models/system/mmeft/spec.yaml

```diff
@@ -0,0 +1,12 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: mmeft
+path: ./
+properties:
+  inference-min-sku-spec: 2|0|7|14
+  inference-recommended-sku: Standard_DS3_v2
+  model_id: mmeft
+tags:
+  Preview: ''
+  license: msft-mit
```

Comment:
why msft-mit?

Reply:
In the link that you had shared about licenses, Microsoft version of MIT is little different from raw MIT license

Reply:
That's still MIT license, right? 
I would recommend keeping MIT as license - same as how we do it for HF and MMD models.